EX_5/src/pepse/PepseGameManager.java

```diff
@@ -6,23 +6,39 @@
 import danogl.gui.SoundReader;
 import danogl.gui.UserInputListener;
 import danogl.gui.WindowController;
+import danogl.util.Vector2;
 import pepse.world.Sky;
+import pepse.world.Terrain;
 import pepse.world.daynight.Night;
 import pepse.world.daynight.Sun;
 
 public class PepseGameManager extends GameManager {
 
     private static float CYCLE_LENGTH = 100;
 
+    //TODO: remove constructor when done.
```

Comment:
you dont need a constructor, if you just call PepseGameManager.run() it will just construct with default size of fullScreen